client/src/partials/patients/documents/modals/documents.modal.html

```diff
@@ -30,28 +30,34 @@
         ng-bind="$ctrl.file.progress + '%'"></div>
     </div>
 
-    <!-- dismiss button  -->
-    <div ng-show="$ctrl.file" class="form-group">
-      <button
-        class="btn btn-danger btn-sm"
-        ng-click="$ctrl.dismiss($ctrl.file)"
-        >
-        <span class="glyphicon glyphicon-trash"></span> {{ 'FORM.BUTTONS.DISMISS' | translate }}
-      </button>
-      {{ $ctrl.file.name }}
-    </div>
+    <div style="margin-top: 10px;" class="row">
+      <div class="form-group col-xs-10"
```

Comment:
This `col-xs-10` breaks on really small devices (like the Nexus 5).  You can try it in your dev tools.

I propose you set two col widths: one for large devices and one for small.  Like this: `col-md-10 col-xs-7`.  That way, the buttons do not overflow on a small device.